bigbone-rx/src/main/kotlin/social/bigbone/rx/RxSuggestionMethods.kt

```diff
@@ -0,0 +1,40 @@
+package social.bigbone.rx
+
+import io.reactivex.rxjava3.core.Completable
+import io.reactivex.rxjava3.core.Single
+import social.bigbone.MastodonClient
+import social.bigbone.api.entity.Suggestion
+import social.bigbone.api.method.SuggestionMethods
+import social.bigbone.rx.extensions.onErrorIfNotDisposed
+
+/**
+ * Reactive implementation of [SuggestionMethods].
+ * Allows access to API methods with endpoints having an "api/vX/suggestions" prefix.
+ * @see <a href="https://docs.joinmastodon.org/methods/suggestions/">Mastodon suggestions API methods</a>
+ */
+class RxSuggestionMethods(client: MastodonClient) {
+
+    private val suggestionMethods = SuggestionMethods(client)
+
+    fun getSuggestions(limit: Int? = null): Single<List<Suggestion>> {
```

Comment:
Please add `JvmOverloads` so that proper Java versions are created with the default arguments applied here.

```suggestion
    @JvmOverloads
    fun getSuggestions(limit: Int? = null): Single<List<Suggestion>> {
```